src/store/application/ui/editPlaylistModal/selectors.ts

```diff
@@ -0,0 +1,20 @@
+import { AppState } from 'store/types'
+import { getCollection as getCollectionById } from 'store/cache/collections/selectors'
+import { getUser as getUserById } from 'store/cache/users/selectors'
+
+const getBase = (state: AppState) => state.application.ui.editPlaylistModal
+export const getIsOpen = (state: AppState) => {
+  return getBase(state).isOpen
+}
+
+export const getCollectionId = (state: AppState) => {
+  return getBase(state).collectionId
+}
+
+export const getCollectionAndUser = (state: AppState) => {
```

Comment:
i'd love us to standardize this selector in the cache or something like that -- we already have some things defined like `getTracksFromCollection` etc.